src/set_options.ts

```diff
@@ -46,6 +46,7 @@ let options: object = {
   language: process.env.FOUNDRY_LANGUAGE || LANGUAGE,
   localHostname: process.env.FOUNDRY_LOCAL_HOSTNAME || null,
   minifyStaticFiles: process.env.FOUNDRY_MINIFY_STATIC_FILES == "true",
```

Comment:
`minifyStaticFiles` is no longer used by Foundry.  It looks like it was renamed to `compressStatic`.  I think we can remove the old option name entirely.

```suggestion
```

Reply:
Build 285 was just released so I'm expediting this change so I can make a single release.

Reply:
Thanks! I had left it in originally, since I didn't know what sort of back compatibility you wanted to maintain.

Reply:
Yup.  That's what I figured.  Thanks for thinking about that.  
If we need to backport anything we can revive the `9.x` branch.
Thank you again.  Your PR made it into the release just a few minutes ago:
- https://github.com/felddy/foundryvtt-docker/releases/tag/v10.285.0